aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -2,19 +2,17 @@
 remote_endpoint: https://fullnode.mainnet.aptoslabs.com
 # replace with below for actual release, compat test needs concrete URL above:
 # remote_endpoint: ~
-name: "vX.YY.Z"
+name: "v1.32.0"
 proposals:
   - name: proposal_1_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework, version vX.YY.Z"
-      description: "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-vX.YY.Z"
+      title: "Multi-step proposal to upgrade mainnet framework, version v1.32.0"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.32"
     execution_mode: MultiStep
     update_sequence:
       - Gas:
-          new: current
-          # replace with below for actual release, above "current" is needed for compat tests:
```

Comment:
is this comment also supposed to be deleted?

Reply:
Yeah, it should be fine (e.g., we did this for 1.31: https://github.com/aptos-labs/aptos-core/pull/16832/files)